### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Responsive

## Trello Card
https://trello.com/c/JhZ650uv/15-dise%C3%B1o-responsive

## Screenshots
captura de pantalla 2018-10-29 a la s 15 58 41

### CODE REVIEW

File: maquetado/scss/footer.scss [hunk 1]
```diff
@@ -21,3 +21,24 @@
 .footer-wolox-icon {
   margin-bottom: 14px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .footer-text {
+    align-self: center;
+    color: $white;
```
Comment (henryzarza) [inline maquetado/scss/footer.scss]: You don't need this

File: maquetado/scss/footer.scss [hunk 2]
```diff
@@ -21,3 +21,24 @@
...
+    font-size: $text-small;
```
Comment (henryzarza) [inline maquetado/scss/footer.scss]: The same here

File: maquetado/scss/footer.scss [hunk 3]
```diff
@@ -21,3 +21,24 @@
...
+  }
+
+  .footer-wolox-icon {
+    margin-bottom: 50px;
+  }
+
+  .footer {
+    align-items: center;
```
Comment (henryzarza) [inline maquetado/scss/footer.scss]: You only need the `padding` property, delete the rest

File: maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -18,3 +18,9 @@
 .nav-title {
   font-size: $text-big;
 }
+
+@media only screen and (max-width: 1024px) {
+  .nav-title {
+    font-size: 0;
```
Comment (henryzarza) [inline maquetado/scss/navbar.scss]: Change this, use `display` property

File: maquetado/scss/styles.scss [hunk 5]
```diff
@@ -1,5 +1,9 @@
 .content {
-  padding: 80px 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  justify-content: center;
+  padding: 100px;
 }
 
 body {
```
Comment (henryzarza) [inline maquetado/scss/styles.scss]: Move this above of `.content` class definition

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -39,20 +38,46 @@
...
+    border-right: 2px solid $green;
+    grid-area: 1/1/3/2;
+    justify-content: center;
+    padding: 0 100px;
+    width: auto;
+  }
+}
+
+@media only screen and (max-width: 1024px) {
+  .tech-container {
```
Comment (henryzarza) [inline maquetado/scss/tech.scss:50]: You only need `grid-template-columns` property

File: maquetado/scss/tech.scss [hunk 7]
```diff
@@ -39,20 +38,46 @@
...
-    border-radius: 10px;
-    content: '';
-    height: 100%;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    transition: transform 0.5s ease;
-    width: 100%;
-    z-index: -1;
+  .tech {
```
Comment (henryzarza) [inline maquetado/scss/tech.scss:54]: You only need `border-bottom` property delete the rest

File: maquetado/scss/tech.scss [hunk 8]
```diff
@@ -39,20 +38,46 @@
...
+    border-bottom: 3px solid $green;
+    display: flex;
+    flex-direction: column;
+    margin: 20px 15px;
+    padding: 20px;
+    position: relative;
+    text-align: center;
+    width: 230px;
+
+    &:first-child {
```
Comment (henryzarza) [inline maquetado/scss/tech.scss:59]: The same here but with `border-right` property

**Review by henryzarza — CHANGES_REQUESTED** [review]

File: maquetado/scss/styles.scss [hunk 9]
```diff
@@ -1,7 +1,11 @@
...
-}
-
 body {
   font-family: 'Roboto', 'sans-serif';
 }
+
+.content {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
```
Comment (henryzarza) [inline maquetado/scss/styles.scss]: Remember the correction of Gabo

File: maquetado/scss/tech.scss [hunk 10]
```diff
@@ -20,17 +20,16 @@
 }
 
 .tech-container {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  align-self: center;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: I think that you don't need this

File: maquetado/scss/tech.scss [hunk 11]
```diff
@@ -20,17 +20,16 @@
...
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Apply the corrections of Grid

File: maquetado/scss/tech.scss [hunk 12]
```diff
@@ -20,17 +20,16 @@
...
+  grid-template-rows: 1fr 1fr;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: The same here

File: maquetado/scss/tech.scss [hunk 13]
```diff
@@ -45,3 +45,23 @@
...
+@media only screen and (max-width: 1024px) {
+  .tech-container {
+    grid-template-columns: 1fr;
+  }
+
+  .tech {
+    border-bottom: 3px solid $green;
+
+    &:first-child {
+      border-right: 0 solid $green;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Change this for `border-right: none`

File: maquetado/scss/tech.scss [hunk 14]
```diff
@@ -45,3 +45,23 @@
...
+      padding: 0;
+      width: 230px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Do you need change this property?

File: maquetado/scss/tech.scss [hunk 15]
```diff
@@ -45,3 +45,23 @@
...
+    }
+
+    &:last-child {
+      border-bottom: 0;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Change this for `border-bottom: none`

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**